app/models/saved_recipie.rb

```diff
@@ -0,0 +1,10 @@
+class SavedRecipie < ApplicationRecord
+  ## VALIDATIONS
+    validates :user_id, presence: true
+    validates :api_recipie_id, presence: true
+    validates :favorited, presence: true
```

Comment:
While looking at rails valiadators I ran acrosss this which I had forgotten so we might want to account for 
```
Since false.blank? is true, if you want to validate the presence of a boolean field you should use one of the following validations:

validates :boolean_field_name, inclusion: [true, false]
validates :boolean_field_name, exclusion: [nil]
Copy
By using one of these validations, you will ensure the value will NOT be nil which would result in a NULL value in most cases.
```
https://guides.rubyonrails.org/active_record_validations.html#presence:~:text=Since%20false.blank,in%20most%20cases.